artichoke-backend/src/extn/core/time/trampoline.rs

```diff
@@ -180,37 +296,46 @@ pub fn to_array(interp: &mut Artichoke, time: Value) -> Result<Value, Error> {
 
 // Math
 
-pub fn plus(interp: &mut Artichoke, time: Value, other: Value) -> Result<Value, Error> {
-    let _ = interp;
-    let _ = time;
-    let _ = other;
-    Err(NotImplementedError::new().into())
+pub fn plus(interp: &mut Artichoke, mut time: Value, mut other: Value) -> Result<Value, Error> {
+    let time = unsafe { Time::unbox_from_value(&mut time, interp)? };
+    if unsafe { Time::unbox_from_value(&mut other, interp) }.is_ok() {
+        Err(TypeError::with_message("time + time?").into())
```

Comment:
```suggestion
        // ```console
        // [3.1.2] > Time.now + Time.now
        // (irb):15:in `+': time + time? (TypeError)
        //         from (irb):15:in `<main>'
        //         from /usr/local/var/rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/irb-1.4.1/exe/irb:11:in `<top (required)>'
        //         from /usr/local/var/rbenv/versions/3.1.2/bin/irb:25:in `load'
        //         from /usr/local/var/rbenv/versions/3.1.2/bin/irb:25:in `<main>'
        // ```
        Err(TypeError::with_message("time + time?").into())
```